app/javascript/packages/document-capture/components/documents-step.jsx

```diff
@@ -52,29 +53,20 @@ function DocumentsStep({
           </p>
         </>
       )}
-      {serviceProvider.name && isMobile && (
-        <p>
-          {formatHTML(
-            t('doc_auth.info.no_other_id_help_bold_html', { sp_name: serviceProvider.name }),
-            {
-              strong: 'strong',
-              a: ({ children }) =>
-                serviceProvider.failureToProofURL ? (
```

Comment:
Need to double-check, but I _think_ this will always be non-empty, based on...

https://github.com/18F/identity-idp/blob/767d81341214f23cd93f78adaaef56e25a9d7e44/app/services/sp_return_url_resolver.rb#L16-L18

Reply:
If tests are any indication, we can expect it to always be a usable string.

https://github.com/18F/identity-idp/blob/9dca42c271721886203ec5e79f7d4358a514b161/spec/services/sp_return_url_resolver_spec.rb#L80-L123